readme.md

```diff
@@ -83,13 +83,13 @@ Your `package.json` will then look like this:
 
 ```json
 {
-  "name": "awesome-package",
-  "scripts": {
-    "test": "ava"
-  },
-  "devDependencies": {
-    "ava": "^0.17.0"
-  }
+	"name": "awesome-package",
+	"scripts": {
+		"test": "ava"
+	},
+	"devDependencies": {
+		"ava": "^0.17.0"
+	}
```

Comment:
Could you undo the whitespace changes in this file? Your editor may be doing it automatically but the only change that should be here is removing the `--source` flag.